packages/ember-metal/lib/utils.js

```diff
@@ -258,6 +258,24 @@ var sourceAvailable = (function() {
   return this;
 }).toString().indexOf('return this;') > -1;
 
+function K(){}
+K.__hasSuper = false;
+
+function checkHasSuper(func) {
+  if (sourceAvailable) {
+    func.toString().indexOf('_super') > -1;
```

Comment:
Should this have a `return`?  Not sure what it would be doing (other than always `return true`) without.